project.yaml

```diff
@@ -22,3 +22,21 @@ actions:
         per_week_pvf: output/code_use_per_week_post_viral_fatigue.csv
         code_table: output/all_long_covid_codes.csv
         practice_summ: output/practice_summ.txt
+
+  generate_cohort_v2:
+    run: cohortextractor:latest generate_cohort --study-definition study_definition_v2
```

Comment:
I guess this will want to be `cohortextractor-v2` fairly soon as that's now a real thing thanks to Ben.

Reply:
Yes, I wasn't sure what the actual new command would be

Reply:
It's alive! https://github.com/opensafely-core/base-docker/pkgs/container/cohortextractor-v2

Although you still won't be able to use it until we update job-runner.